lib/rubocop/cop/betterment/authorization_in_controller.rb

```diff
@@ -41,13 +41,8 @@ def initialize(config = nil, options = nil)
         end
 
         def on_class(node)
-          Utils::MethodReturnTable.populate_index node
-          Utils::MethodReturnTable.indexed_methods.each do |method_name, method_returns|
-            method_returns.each do |x|
-              name = Utils::Parser.get_root_token(x)
-              @param_wrappers << method_name if name == :params || @param_wrappers.include?(name)
-            end
-          end
+          @class_methods = Utils::Parser.get_instance_methods(node)
+          @param_wrappers.concat(find_param_wrappers(@class_methods))
```

Comment:
__TODO:__ Change this to *not* set `@class_methods` or `@param_wrappers` anymore.